modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/sidebar/DataAndViewsTab.es.js

```diff
@@ -477,6 +541,7 @@ export default function DataAndViewsTab({
 							</label>
 
 							<SelectTableView
+								addButton={addTableViewButton(updateTableView)}
```

Comment:
Hey @gabrielpramos, I think that at this case we don't need pass the selectTableView callback because we just have only one and we always call the same callback.

Reply:
You can just call the updateTableView inside the openTableViewModal.

Reply:
yeah, sure :D